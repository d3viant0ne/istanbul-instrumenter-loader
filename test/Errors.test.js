import loader from '../src';

describe('Errors', () => {
  test('Options Validation Error', () => {
    const err = () => loader.call({ query: { debug: 'foo' } });

    expect(err).toThrow();
    expect(err).toThrowErrorMatchingSnapshot();
  });
});
