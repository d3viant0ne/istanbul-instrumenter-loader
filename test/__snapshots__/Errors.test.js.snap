// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors Options Validation Error 1`] = `
"Validation Error

Istanbul Instrumenter Loader Invalid Options

options.debug should be boolean
"
`;
